src/Microsoft.Identity.Client/AppConfig/AbstractApplicationBuilder.cs

```diff
@@ -230,10 +253,10 @@ protected T WithOptions(ApplicationOptions applicationOptions)
         /// </summary>
         /// <param name="component">identifier of the software component (libraries/SDK) consuming MSAL.NET</param>
         /// <returns>The builder to chain the .With methods</returns>
+        [Obsolete("Should use ClientName and ClientVersion properties instead of Component", true)]
         public T WithComponent(string component)
```

Comment:
Should it be in Migration Aid ?
or otherwise debugger hidden?

Reply:
Or just remove them, i reckon there are very few developers using this (possibly 1-2 internal ones?)

Reply:
I've just deleted WithComponent then.